app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter
```

Comment:
[ OPTIONAL ] It's a suggestion that you can use Active Record Call back `after_save` to call the `#update_likes_counter` method every time a new post has been saved to database, and make the `update_likes_counter` method private.